tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -296,6 +296,11 @@ kbs_k8s_svc_host() {
 	if kubectl get ingress -n "$KBS_NS" 2>/dev/null | grep -q kbs; then
 		kubectl get ingress kbs -n "$KBS_NS" \
 			-o jsonpath='{.spec.rules[0].host}' 2>/dev/null
+	elif kubectl get svc kbs-nodeport -n "$KBS_NS" &>/dev/null; then
+			local host
+			host=$(kubectl get nodes -o jsonpath='{.items[0].status.addresses[?(@.type=="ExternalIP")].address}' -n "$KBS_NS")
+			[ -z "$host"] && host=$(oc get nodes -o jsonpath='{.items[0].status.addresses[?(@.type=="InternalIP")].address}' -n "$KBS_NS")
```

Comment:
Hi @ldoktor !

I looked at the code and I didn't find anything wrong. Then I give a try on my local machine but it failed to return the service address. Luckily I changed of laptop recently and I don't have `oc` installed yet, so that's the problem it is using `oc` here :D

Reply:
Oups, let me fix that (oc is shorter than kubectl...)

Reply:
The new version worked for me @ldoktor !